cmd/plume/upload.go

```diff
@@ -95,17 +110,76 @@ func runUpload(args []string) int {
 		return 1
 	}
 
-	fmt.Printf("Writing %v to %v\n", imageName, bucket)
+	fmt.Printf("Writing %v to gs://%v ...\n", imageNameGS, bucket)
+	fmt.Printf("(Sometimes this takes a few mintues)\n")
+
+	/*
```

Comment:
Recently uploading to GS has gone from going quite fast to taking several minutes. It eventually succeeds.I figured it is important that user's know its not broken when its taking 5 or more minutes to upload ~100-200MB.